src/utils/global/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react'
+
+import styles from './tooltip.module.scss'
+
+interface TooltipProps {
+  text: string;
+  children: React.ReactNode;
+  title: string;
+  textContainerWidth?: string;
+  leftOffset?: string;
+}
+
+const Tooltip: React.FC<TooltipProps> = (props) => {
+  const [show, setShow] = useState(false);
+
+  const tooltipStyle: React.CSSProperties = {
+    opacity: show ? 1 : 0,
+    width: props.textContainerWidth ? props.textContainerWidth : "max-content",
+    left: props.leftOffset,
+  };
+
+  return (
+    <>
```

Comment:
Fragment isn't needed here, as you're returning a single div element as the root